README.md

```diff
@@ -88,13 +89,16 @@ const schemas: ERC725JSONSchema[] = [
 const erc725 = new ERC725(schemas, address, RPC_URL, config);
 ```
 
-### JavaScript
+</details>
+
+<details>
+  <summary>Javascript</summary>
 
 ```js
-import { ERC725 } require('@erc725/erc725.js');
+const { ERC725 } = require('@erc725/erc725.js');
 
-// Part of LSP3-UniversalProfile Schema
-// https://github.com/lukso-network/LIPs/blob/master/LSPs/LSP-3-UniversalProfile.md
+// Part of LSP3-Profile-Metadata Schema
+// https://github.com/lukso-network/LIPs/blob/master/LSPs/LSP-3-Profile-Metadata.md
```

Comment:
FMI: What s the difference btw `LSP3Profile` and `SupportedStandards:LSP3Profile` as a schema?

Reply:
`LSP3Profile` = contains the `VerifiableURI` that links to the JSON file. It is the actual profile metadata

`SupportedStandards:LSP3Profile` = it is just a data key to confirm that this contract supports the LSP3 Metadata standard, and contain a certain set of keys (only `LSP3Profile` in this case).